apps/egghead/src/lib/sanity/resource.ts

```
@@ -0,0 +1,204 @@
+		parseInt(instructorId.toString()),
+	)
+
+	if (!sanityInstructorReference) {
+		throw new Error(`Sanity instructor with id ${instructorId} not found.`)
+	}
+
+	switch (post.fields.postType) {
+		case 'lesson':
+			const sanityLesson = await getSanityLessonForEggheadLessonId(
+				post.fields.eggheadLessonId,
+			)
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)